cala-ledger-core-types/Cargo.toml

```diff
@@ -6,9 +6,12 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+cala-cel-interpreter = { path = "../cala-cel-interpreter/", package = "cala-cel-interpreter" }
```

Comment:
I don't think you need `package = ...` if it's the same. The only reason for `package = ` is if you want to rename it internal to this crate.

Like probably you could simply call it `cel_interpreter` internally....
(check how its done in sqlx-ledger)